py-polars/docs/source/reference/series.rst

```diff
@@ -78,6 +78,7 @@ Descriptive stats
     Series.is_utf8
     Series.len
     Series.n_unique
+    Series.has_validity
```

Comment:
@ritchie46 : Not 100% sure on this one, do we want this to be part of the API?

Reply:
@zundertj 
I meant to write the docstrings like this:


```python
>>> df = pl.DataFrame({
...    "ham": [1, 2, 3],
...    "Hambureger": [11, 22, 33],
...    "foo": [3, 2, 1]
... })
```

Instead of:
```python
>>> df = pl.DataFrame({
... "ham": [1, 2, 3],
... "Hambureger": [11, 22, 33],
... "foo": [3, 2, 1]})
```

Reply:
Yes, I also prefer the 4 spaces indentation. And I believe that's the pep compliant way to fmt it. (Otherwise we should do what black does in our cfg).

Reply:
To be sure, it was already like that, I only did the translation of `>>>` to `...`in first instance. Pushed now a commit to fix it.

Would be great if black can format code examples as well, because it is still not all completely consistent with indentation of closing brackets... A formatter is much better at this than doing this by hand. As far as I know, there is no tool for it, but if anyone knows a way to do it, that would be appreciated.